react-native-gutenberg-bridge/ios/Gutenberg.swift

```diff
@@ -61,6 +70,8 @@ public class Gutenberg: NSObject {
     public init(dataSource: GutenbergBridgeDataSource, extraModules: [RCTBridgeModule] = []) {
         self.dataSource = dataSource
         self.extraModules = extraModules
+        super.init()
+        logThreshold = isPackagerRunning ? .trace : .error
```

Comment:
Can we do `.warn` instead of `.error` our would that still be too verbose?

Reply:
Yeah we can, we would see all the YellowBox messages though. 
And `.error` is the default for for Release builds, so if we change it, we would have to check for `#if DEBUG` to select between `.warn` and `.error`.

As you mentioned, maybe we can leave it as it is now and tweak it later if needed.